### ISSUE TITLE
request: checksum binaries also shown on other websites then github

### ISSUE THREAD

**Issue by wonkytonky** [iotaledger/wallet#566]
recently some bitcoin forks github account got compromised...   you can upload a bad binary and also change the checksum on github info page..  .. so it would be a good idea to show the checksum in your blog/tweets etc..  so ppl can double check.  against github.

**Comment by shday** [comment]
Yeah, having the binaries and the checksums in the same place isn't very useful.

Based on the number of issues here, someone needs to double check 2.5.4.
